runtime/local/Cargo.toml

```diff
@@ -72,6 +72,9 @@ pallet-evm-precompile-xvm = { git = "https://github.com/AstarNetwork/astar-frame
 pallet-precompile-dapps-staking = { git = "https://github.com/AstarNetwork/astar-frame", branch = "polkadot-v0.9.33", default-features = false }
 pallet-xvm = { git = "https://github.com/AstarNetwork/astar-frame", branch = "polkadot-v0.9.33", default-features = false, features = ["evm", "wasm"] }
 
+# pallet-asset chain-extension
```

Comment:
Seems like a redundant comment :slightly_smiling_face: 

Why not add a comment like:
`# Chain extensions`
?

That would indicate to put all chain-extension depps here.